sklearn/base.py

```diff
@@ -437,8 +437,20 @@ def _validate_data(self, X, y=None, reset=True,
 
         return out
 
+    @property
     def _repr_html_(self):
-        """HTML representation of estimator"""
+        """HTML representation of estimator.
+
+        Used to display the HTML representation in sphinx-gallery.
+        Jupyer kernels will use `_repr_mimebundle_`.
```

Comment:
There is a typo:

```suggestion
        Jupyter kernels will use `_repr_mimebundle_`.
```

and apparently this is wrong based on your last comment. I am not sure if `_repr_mimebundle_` will ever take the priority. Maybe we can just state:


This is redundant with the logic of `_repr_mimebundle_`. The latter should be favored in the long term, `_repr_html_` is only implemented for consumer who do not interpret `_repr_mimebundle_`.